lib/reporters/cli/index.js

```diff
@@ -319,7 +321,7 @@ _.assignIn(PostmanCLIReporter, {
         });
 
         // add specific rows to show in summary
-        stats && _.forEach([{
+        run.stats && _.forEach([{
```

Comment:
Instead of changing this at 100 different places, I think we can declare the value at the start and use that. It would be much easier to read.
```suggestion
		var stats = run && run.stats;

		// add specific rows to show in summary
		stats && _.forEach([{
```